08-Message-Types.md

```diff
@@ -28,7 +29,6 @@
 | 0x22                           | 0               | SetCustomMiningJob                 |
 | 0x23                           | 0               | SetCustomMiningJob.Success         |
 | 0x24                           | 0               | SetCustomMiningJob.Error           |
-| 0x25                           | 0               | Reconnect                          |
 | 0x26                           | 0               | SetGroupChannel                    |
```

Comment:
I leave it to 0x26 cause I do not know if braiins is already using this message or not

Reply:
I would ask them and change it to 0x25 if it's not a problem